google-cloud-storage/acceptance/storage/bucket_generate_signed_post_policy_v4_test.rb

```diff
@@ -228,6 +230,47 @@
       _(File.read(downloaded.path, mode: "rb")).must_equal File.read(data, mode: "rb")
     end
   end
+focus
+  it "generates a signed post object v4 with conditions" do
+    conditions = [["starts-with", "$key", ""]]
```

Comment:
Remove the additional `[]` brackets around the condition. This was the issue.

Reply:
Done, thanks!